lib/Common/CommonPal.h

```diff
@@ -554,3 +567,61 @@ void TryFinally(const TryFunc& tryFunc, const FinallyFunc& finallyFunc)
 
     finallyFunc(hasException);
 }
+
+
+namespace PlatformAgnostic
+{
+    __forceinline unsigned char _BitTestAndSet(LONG *_BitBase, int _BitPos)
+    {
+#if defined(__clang__)
```

Comment:
Probably should check for x86 compatible arch? Does clang on windows target arm right now?